### ISSUE TITLE
Empty Visualization when using --pipeline <pipeline_name>

### ISSUE THREAD

**Issue by johnkmaxi** [kedro-org/kedro-viz#725]
## Description
When trying to visualize a specific pipeline, no visualization is created.

## Context
How has this bug affected you? What were you trying to accomplish?
I would like to be able to visualize the various sub-pipelines in my project

## Steps to Reproduce

1. create pipelines 
2. register pipelines
3. run pipelines
4. kedro viz --pipeline <pipeline_name>

## Expected Result
A visual of the named pipeline should appear in the browser window

## Actual Result
kedro viz tab opens with no visualized pipeline. If I do `kedro viz` then my default pipeline shows up fine. If I do `kedro viz --pipeline __default__`, it works. In fact, any pipeline can be visualized if I change it's key in the pipeline registry to '__default__'.
If I run `kedro viz --pipeline <pipeline_name>`, open blank visual in chrome ctrl-c in command window then run `kedro viz` it will show me the previously named pipeline rather than the default pipeline.

```
-- If you received an error, place it here.
```
INFO:     127.0.0.1:49846 - "GET /manifest.json HTTP/1.1" 404 Not Found
```
-- Separate them if you have more than one.
```

## Your Environment
Include as many relevant details as possible about the environment you experienced the bug in:

* Web browser system and version: Chrome Version 97.0.4692.99 (Official Build) (64-bit)
* Operating system and version: Windows Server 2016
* NodeJS version used (if relevant):
* Kedro version used (if relevant): 0.17.6
* Python version used (if relevant): 3.7.1
* Kedro Viz version: 4.2.0

## Checklist
- [ ] Include labels so that we can categorise your issue
- [ ] Bug

**Comment by tynandebold** [comment]
Hey, @johnkmaxi. Thank you for finding and flagging this issue! We'll have a look soon and get back to you.

**Comment by AntonyMilneQB** [comment]
Thank you for the bug report! This should be fixed by #729.
